dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/netty/inbound/NettyTCPWriteRequestContext.java

```diff
@@ -232,6 +234,11 @@ public long write(long numBytes, int timeout) throws IOException {
         long writtenBytes = 0L;
         // If using HTTP2 chunk logic or something else, keep the relevant parts.
         final String protocol = nettyChannel.attr(NettyHttpConstants.PROTOCOL).get();
+
+        // A write queue to run all the write events inside the eventloop to improve performance
+        // TODO Maybe we should see if this writequeue should belong to the class to improve performance?
```

Comment:
This write queue object we should check if this should belong to the class and shared with multiple calls and limit the creation of the object or if it makes sense to leave as is

Reply:
Created https://github.com/OpenLiberty/open-liberty/issues/31555 for this